Tests/Unit/NetworkServer/JsonHandlers/LnsStationConfigurationTests.cs

```diff
@@ -15,8 +15,7 @@ namespace LoRaWan.Tests.Unit.NetworkServerTests.JsonHandlers
     using Xunit;
     using static Bandwidth;
     using static SpreadingFactor;
-    using static NetworkServer.BasicsStation.RouterConfigStationFlags;
```

Comment:
why remove the static imports here? does it not work with `global::` imports?

Reply:
No, I still get a warning with `global::`